web-bluetooth/_includes/value-to-device-type.html

```diff
@@ -1,5 +1,5 @@
 <script>
-var valueToDeviceType = {
+const valueToDeviceType = {
```

Comment:
The Google JS style guide says to use `VALUE_TO_DEVICE_TYPE` for constants. Is that something that should be followed here?